packages/db/lib/utils.js

```diff
@@ -45,8 +46,25 @@ function urlDirname (url) {
   return dirname(fileURLToPath(url))
 }
 
+function locateSchemaLock (configManager) {
+  return configManager.current.db.schemalock.path ?? join(dirname(configManager.fullPath), 'schema.lock')
```

Comment:
Is `configManager.dirname` safe to use instead of calling `dirname()`?

Reply:
I forgot about it.